test/lit/binary/debug-bad-binary.test

```diff
@@ -0,0 +1,42 @@
+# Verify --debug mode prints partial results even after an error.
```

Comment:
I think we generally use `;;` comment syntax even in .test files.

Reply:
Ah, I copied it from https://github.com/WebAssembly/binaryen/blob/main/test/lit/binary/stacky-eh.test - should I change that too?

Reply:
It's not that important. I suspected we might not be consistent here...

Reply:
Ah, already done... a small step towards eventual consistency.